lib/rubocop/cop/mixin/multiline_literal_brace_layout.rb

```diff
@@ -69,13 +69,18 @@ def check_symmetrical(node)
       end
 
       def ignored_literal?(node)
-        implicit_literal?(node) || empty_literal?(node) || node.single_line?
+        implicit_literal?(node) || empty_literal?(node) || single_line?(node)
       end
 
       def implicit_literal?(node)
         !node.loc.begin
       end
 
+      def single_line?(node)
+        return true if node.single_line?
```

Comment:
I don't like the fix that much. Seems we're simply not checking the right node here...